tox.ini

```diff
@@ -1,41 +1,40 @@
 [tox]
+setenv=PYTHONDONTWRITEBYTECODE=1
 skipsdist=True
 envlist =
-    py{27,33,34,35}-{test,lint},
+    py{34,35}-{test,lint},
```

Comment:
I'll accept this as-is, but I have changes that I'm going to make around the tox.ini/setup.cfg files, etc, once I get typingplus out.